styleguide/_themes/global/scss/tables.scss

```diff
@@ -0,0 +1,105 @@
+$table-padding: 20px 0;
+
+.productTable {
+  border-collapse: collapse;
+  width: 100%;
+}
+
+.productTable-feature,
+.productTable-featureItem,
+.productTable-name {
+  border: 1px solid $light-gray;
+}
+
+.productTable-name {
+  @include heading;
+  background-color: $darkteal;
+  color: $white;
+  padding: $table-padding;
+  text-align: center;
+}
+
+.productTable-feature {
+  @include subhead;
+  color: $darkteal;
+  padding: $table-padding;
+  text-align: center;
+}
+
+.productTable-featureItem {
+  &::before {
+    @include center-block;
+    font-family: FontAwesome;
+    padding: 10px 0;
+    text-align: center;
+  }
+}
+
+.offered {
+  &::before {
+    color: $darkteal;
+    content: '\f00c';
+  }
+}
+
+.notOffered {
+  &::before {
+    color: $light-gray;
+    content: '\f00d';
+  }
+}
+
+@media screen and (max-width: 768px) {
+  .productTable {
+    display: flex;
+    flex-flow: column;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  .productTable-row {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-around;
+  }
+
+  .productTable-name {
+    display: flex;
+    flex-flow: row;
+    flex-grow: 1;
+    font-size: .70em;
+    justify-content: center;
+
+    :first-child {
+      display: none;
+    }
+  }
+
+  .productTable-feature,
+  .productTable-featureItem {
+    display: flex;
+    flex-flow: row;
+    flex-grow: 1;
+    justify-content: center;
+
+    :first-child {
+      background: $light-gray;
+      display: block;
+      width: 100%;
+    }
+  }
+
+  .horizontal-scroll {
```

Comment:
@amberRrucker Definitely makes sense. Like I said, I haven't fully processed the concept, but I think it at least gives us an option, but I'd be interested in what the others have to say about this.